src/test/resources/scripts/init.sql

```diff
@@ -31,6 +31,7 @@ CREATE TABLE IF NOT EXISTS Consignment (
   TransferInitiatedBy uuid,
   ExportDatetime timestamp with time zone,
   ExportLocation text,
+  ExportVersion text,
```

Comment:
This is the script for the old H2 database which we shouldn't need any more. I  meant to delete it. You can delete it as part of this PR or leave it for another time but you don't need to add the field in.